nginx-redirects.conf

```diff
@@ -293,8 +293,15 @@ server {
 
 server {
     include /etc/nginx/ssl.default.conf;
+    include /etc/nginx/api-proxy.conf;
     server_name diskdetective.org www.diskdetective.org;
-    return 301 https://www.zooniverse.org/projects/ssilverberg/disk-detective;
+
+    location /subjects/ {
+        return 301 https://static.zooniverse.org/$host$uri;
+    }
+
+    location / {
+        return 301 https://www.zooniverse.org/projects/ssilverberg/disk-detective;
```

Comment:
```suggestion
        return 301 https://www.zooniverse.org/projects/ssilverberg/disk-detective;
    }
```

Reply:
Yep -- caught it from the Jenkins output just before you posted here.